extra-multimedia/feeluown-local/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME="feeluown-local"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for the local music source(s)"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.